cl/lib/zoho.py

```diff
@@ -0,0 +1,100 @@
+from django.conf import settings
+from django.core.cache import cache
+from zohocrmsdk.src.com.zoho.api.authenticator import OAuthToken
+from zohocrmsdk.src.com.zoho.crm.api import (
+    HeaderMap,
+    Initializer,
+    ParameterMap,
+)
+from zohocrmsdk.src.com.zoho.crm.api.record import (
+    APIException,
+    RecordOperations,
+    ResponseWrapper,
+    SearchRecordsParam,
+)
+
+
+def get_zoho_cache_key() -> str:
+    return "zoho_token"
+
+
+class SearchableModule:
+    module_name: str = ""
+
+    @staticmethod
+    def initialize():
+        refresh_token = cache.get(f"{get_zoho_cache_key()}:refresh")
+        if not refresh_token:
+            raise Exception(
+                f"Cache miss: no value found for key {get_zoho_cache_key()}:refresh"
```

Comment:
We can tell people to run the script in this case, right?

Reply:
Oh, you’re right! We can definitely improve the error message.